app/models/attachment.rb

```diff
@@ -1,15 +1,19 @@
 require_dependency 's3_url_signer'
 
 class Attachment < ActiveRecord::Base
+  THUMBNAIL_SIZES = { thumb: [600, 400], tile: [118, 154] }
+
   belongs_to :attachable, polymorphic: true
   belongs_to :story
   belongs_to :owner, class_name: "User"
 
   validates_presence_of :owner_id
 
+  after_create :process_upload
+
   def self.all_for_user(user)
     users_story_ids = Story.where("stories.project_id IN (?)", user.project_ids).pluck(:id)
-    where("attachments.story_id IN (?)", users_story_ids).order(created_at: :desc)
+    where("attachments.story_id IN (?) AND attachments.attachable_id IS NOT NULL", users_story_ids).order(created_at: :desc)
```

Comment:
Would revert this as well, just because it’s not attached to anything doesn’t mean it shouldn’t show up in the `/files` page